inc/Gameboard.hpp

```diff
@@ -1,11 +1,12 @@
 #pragma once
+
 #include <memory>
-#include "RandomGeneratorI.hpp"
 
-using RandomGeneratorPtr = std::shared_ptr<RandomGeneratorI>;
+#include "RandomGenerator.hpp"
 
+using RandomFromRangeGeneratorPtr = std::shared_ptr<RandomFromRangeGeneratorI>;
```

Comment:
I think better name is RandomRangeGenerator

Reply:
Done